packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -10,8 +10,6 @@ import { initAPIDiscovery } from './api-discovery';
 import { useReduxStore } from './redux';
 
 type AppInitSDKProps = {
-  /** Only child is your Application */
-  children: React.ReactElement;
```

Comment:
`React.FC` covers this... not sure why we overrode it 🤔 

Causes a weird call: https://github.com/openshift/hac-core/pull/43/files#diff-407ff414eb218ddfa29e668c31dfa67956c70968492131cf233a735742ca1addR16

Reply:
I could've sworn this has come up before and I removed `children` then added it back... but now I can't find the comment. I'm fine removing it.

Reply:
Right, well I think we wanted to control the flow -- but this makes it very difficult for others to use the API. I guess if we want this as a stance, I can modify the props in my component to only accept one and thus thread that logic forward... but I'm wondering why we did it as there is nothing I can think of that won't work if we just do it this way.

If I recall correctly, there was a lint error on passing children without Fragments... but it wasn't necessarily a "design choice" so much as a "fix for the eslint disable"